core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,18 +18,59 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
+    @media(max-width: 768px) {
+      /* This prevents the navbar from collapsing in the exploration page,
+      so that the exploration title can be shown. */
+      .oppia-exploration-header {
+        display: block;
+        float: right;
+        margin-top: 2px;
+        max-height: 2ch;
+        max-width: 22ch;
+        overflow: hidden;
+      }
+      .oppia-navbar-collapse.collapse {
+        display: block;
+      }
+    }
+    @media(max-width: 600px) {
+      .oppia-exploration-header {
+        max-width: 16ch;
+      }
+      .oppia-logo {
+        height: 48px;
+        margin-top: 8px;
+        margin-left: 0;
+        max-width:none;
+      }
+      .oppia-navbar-brand-name {
+        max-width: 35px;
+        overflow: hidden;
+      }
+      .oppia-navbar-breadcrumb-icon {
+        display: none;
+      }
+   }
+   @media(max-width: 450px) {
+      .oppia-exploration-header {
```

Comment:
This and the next two lines should be indented by 2 from the previous line, not 3.